src/components/cloud-cover/cloud-cover.scss

```diff
@@ -195,6 +196,10 @@ $_gap: ms.step(1);
 
 .c-cloud-cover__content {
   grid-area: content;
+
+  @media (width > breakpoint.$l) {
+    font-size: size.$font-big;
+  }
```

Comment:
We're trying to avoid "hard" jumps like this in the new design in favor of gradual, fluid sizing increases.

So instead of a media query like this, we would add this to the start of the `.scss` file:

```scss
@use '../../mixins/fluid';
```

And then in `.c-cloud-cover__content`, we would use:

```scss
.c-cloud-cover__content {
   // ...
    font-size: fluid.fluid-clamp(
      1em,
      size.$font-big,
      breakpoint.$xs,
      breakpoint.$l
    );
}
```

This will cause the font size to gradually increase from `1em` to the value of `size.$font-big`.